src/main/java/swift/ui/MainWindow.java

```diff
@@ -180,40 +181,45 @@ private void handleExit() {
      */
     @FXML
     private void handleTaskTab() {
```

Comment:
the method name can be more clearer :)
```suggestion
    private void showTaskTab() {
```
handle doesn't really describe what it does